lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -67,7 +67,7 @@
    - mysql (command line binary)
    - mysqldump (command line binary)
 notes:
-   - Requires the python-mysqldb package on the remote host, as well as mysql and mysqldump binaries.
+   - Requires the PyMySQL package on the remote host, as well as mysql and mysqldump binaries.
```

Comment:
I think it will be confusing that the `module.fail_json` then says that `MySQL-python` is one requirement you can choose after reading this. It might be useful to say here something to the effect of: 'you are encouraged to use PyMySQL for all the benefits it brings but you can still use MySQL-python but it will be deprecated at some point'. The module is in transition to a better supported dependency and I think it is OK to say that up front.